### ISSUE TITLE
ProgressEvent data is weird

### ISSUE THREAD

**Issue by connorshea** [mdn/browser-compat-data#2293]
So in #1532 [`ProgressEvent.json`](https://github.com/mdn/browser-compat-data/blob/c21485856b6b2aa2effd9b7a37e281bd3c108fab/api/ProgressEvent.json) was added, albeit empty with no data (and also missing the `ProgressEvent()` constructor).

There are a few problems with this:

- The data in the compatibility tables for each page is really weird and inconsistent, e.g. [`total`](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent/total) and [`loaded`](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent/loaded) have "No support" in Chrome but the parent [`ProgressEvent`](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent) page is listed as being supported from v1 of Chrome. [`lengthComputable`](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent/lengthComputable) has Chrome support starting at v50.
- The spec changed at some point and every browser seemed to have replaced(?) `initProgressEvent()` with the `ProgressEvent()` constructor.

**TL;DR** someone who has a better understanding of Chromium/WebKit/Firefox needs to research this and figure out what the actual support for each browser is.

Notes/Sources:
- Based on [Chrome Status](https://www.chromestatus.com/feature/5044837464145920) I think the feature existed under the name `XMLHttpRequestProgressEvent` in WebKit, but was replaced by `ProgressEvent` in Chrome 50.
- You can see its implementation in [the Chromium source](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/events/progress_event.h?q=ProgressEvent&sq=package:chromium&g=0&l=35).
- There was a [WebKit bug to remove `initProgressEvent`](https://bugs.webkit.org/show_bug.cgi?id=71340), it was replaced by the `ProgressEvent()` constructor in the standard.
- [Firefox bug for removing `initProgressEvent()`](https://bugzilla.mozilla.org/show_bug.cgi?id=843489)

**Comment by caugner** [comment]
Ah, sorry, I didn't see your issue. I should incorporate the info you found into #2300.

> There was a WebKit bug to remove initProgressEvent, it was replaced by the ProgressEvent() constructor in the standard.

I think `initSomeEvent` was the old pattern to create events in the past, and was first replaced by `document.createEvent('SomeEvent')` and then by constructors, though not all events are necessarily migrated yet. Probably it's desirable to document both when the constructor were first supported and the `initSomeEvent` last supported.

**Comment by connorshea** [comment]
@caugner you might want to add WIP to the PR title so it doesn't get merged, just in case :) I'm not sure how to handle this situation, but the current tables on MDN seem incorrect to me at first glance.

**Comment by connorshea** [comment]
The PR was merged so I think this can be closed now :)
